frontend/src/views/Lobby.tsx

```diff
@@ -227,9 +227,10 @@ function LobbyPage() {
       history.push('/game', {
         roomId: currentRoomId,
         currentUser,
+        difficulty,
```

Comment:
Yeah that makes sense - I'm not sure if that's implemented right now but I can see in the future if on the backend the game object is created and automatically assigned a random problem from there, which is then sent to the frontend on the game page (that would also help to prevent problems from changing on refresh). But yeah lmk if you think that I should do that in this or the next PR

Reply:
You can do that in the next PR, I see no need now since we haven't implemented the game fetch logic yet -- just wanted to note this down. 👍